jekyll/_includes/snippets/runner/launch-agent-download.adoc

```diff
@@ -0,0 +1,109 @@
+// Steps to download and run launch agent script
+
+
+ifndef::windows[]
+// Display the download-launch-agent step only for macOS and Linux
+. Save the https://raw.githubusercontent.com/CircleCI-Public/runner-installation-files/main/download-launch-agent.sh[download-launch-agent script] in the root of your project. When run (see the next step) the script will download the launch-agent binary, verify the checksum, and set the launch agent version.
++
+Self-hosted runners on cloud auto-update to the latest supported versions. For server, specific self-hosted runner versions are validated for interoperability, and self-hosted runners do not auto-update.
+
+. Set your target platform and run the `download-launch-agent.sh` script to download, verify, and install the binary. If you are using **cloud**, use the table below to find your platform variable:
++
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Installation Target
+| Variable
+
+// Display only Linux target platform options on the Linux install page
+ifdef::linux[]
+| Linux x86_64
+| `platform=linux/amd64`
+
+| Linux ARM64
+| `platform=linux/arm64`
+
+| Linux s390x
+| `platform=linux/s390x`
+
+| Linux ppc64le
+| `platform=linux/ppc64le`
+endif::[]
+
+// Display only macOS target platform options on the macOS install page
+ifdef::macos[]
+| macOS x86_64
+| `platform=darwin/amd64`
+
+| macOS M1
+| `platform=darwin/arm64`
+endif::[]
+|===
+
+ifdef::linux[]
++
+// Display code snippet for Linux installation only
+For example, on **cloud**, to set your platform for Linux x86_64 and run the `download-launch-agent.sh` script, run the following:
++
+```shell
+export platform=linux/amd64 && sh ./download-launch-agent.sh
+```
+endif::[]
+
+ifdef::macos[]
++
+// Display code snippet for macOS installation only
+For example, on **cloud**, to set your platform for macOS M1 and run the `download-launch-agent.sh` script, run the following:
++
+```shell
+export platform=darwin/arm64 && sh ./download-launch-agent.sh
+```
++
+endif::[]
+// The following closes the ifndef on line 4
+endif::[]
+
+. For *server v3.1.0 and up*, use the table below to find the compatible machine runner launch-agent version for the version of server you are running:
```

Comment:
One issue here is the order of the steps is going wrong, this is shown as `1.` , but actually I don't think this needs to be a next step as it's just an alternative to what's described for cloud above, so maybe just use a `+z for now to keep all that content together in one step? Cloud/Server tabs might be nice here but I think that might just get too complicated?

Reply:
@rosieyohannan Unfortunately I wasn't able to figure out a formatting that would work well for Windows! Linux/macOS look OK. Any ideas?

Reply:
I have just pushed an addition of a bullet point for Windows to "attach" to the rest of the content. It's not really needed obviously but it fixes the flow. Might be a least-worst option?